problem_builder/public/js/mentoring_with_steps.js

```diff
@@ -83,12 +83,18 @@ function MentoringWithStepsBlock(runtime, element) {
         if (response.step_status === 'correct') {
             checkmark.addClass('checkmark-correct icon-ok fa-check');
             checkmark.attr('aria-label', checkmark.data('label_correct'));
+            checkmark.find('span.sr-only').remove();
+            checkmark.append('<span class="sr-only">'+gettext("Correct")+'</span>')
         } else if (response.step_status === 'partial') {
             checkmark.addClass('checkmark-partially-correct icon-ok fa-check');
             checkmark.attr('aria-label', checkmark.data('label_partial'));
+            checkmark.find('span.sr-only').remove();
+            checkmark.append('<span class="sr-only">'+gettext("Partially correct")+'</span>')
         } else {
             checkmark.addClass('checkmark-incorrect icon-exclamation fa-exclamation');
             checkmark.attr('aria-label', checkmark.data('label_incorrect'));
+            checkmark.find('span.sr-only').remove();
```

Comment:
Could we extract it before the `if` instead of repeating it three times?

Reply:
Yes I am Accommodating this. Thank you for pointing.